### PR TITLE
Switch to Xcode 13.2.1 in `swift.yml` workflow

### PR DESCRIPTION
**Pull request by MaxDesiatov**
Currently selected version of Xcode makes PRs fail on CI. Let's switch to the latest one. I've also updated SwiftPM dependencies for compatibility with Xcode 13.

### CODE REVIEW

File: Tests/webidl2swiftTests/webidl2swiftTests.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import XCTest
 import ArgumentParser
 import Commands
-@testable import webidl2swift
+@testable import Commands
```
Comment (j-f1) [inline Tests/webidl2swiftTests/webidl2swiftTests.swift:8]: Why this change?

**Review by j-f1 — APPROVED** [review]
LGTM if it works on your end, although I’m curious about this change ⬇️

File: Tests/webidl2swiftTests/webidl2swiftTests.swift (hunk 1 above)
Comment (MaxDesiatov) [reply Tests/webidl2swiftTests/webidl2swiftTests.swift:8]: Latest Xcode complains about `webidl2swift` being an executable module, it says such modules can no longer be imported in test targets. So I'm importing `Commands`, which is a library target.

Comment (j-f1) [reply Tests/webidl2swiftTests/webidl2swiftTests.swift:8]: Ok, cool!
